taipy/core/cycle/cycle.py

```diff
@@ -36,6 +60,42 @@ class Cycle(_Entity, _Labeled):
         end_date (datetime): The date and time of the end of this cycle.
         name (str): The name of this cycle.
         properties (dict[str, Any]): A dictionary of additional properties.
+
+    !!! example "Example for January cycle"
+
+        ![cycles](../refmans/img/cycles_january_colored.svg){ align=left width="250" }
+
+        Let's assume an end-user publishes production orders (i.e., a production plan) every
+        month. During each month (the cycle), he/she will be interested in "playing" with
+        different scenarios until only one of those scenarios is selected as the official
+        production plan to be published. Each month is modeled as a cycle, and each cycle
+        can contain one or more scenarios.
+
+        The picture on the left shows the tree of entities: Cycles, Scenarios, and their
+        associated Sequence(s). There is an existing past cycle for December and a current
+        cycle for January containing a single scenario.
+
+    When comes the end of a _cycle_ (start date + duration), only one of the scenarios is
+    applied in production. This "official" scenario is called the _**primary scenario**_.
+    Only one _**primary scenario**_ per cycle is allowed.
+
+    !!! example "Example for February cycle"
+
+        ![cycles](../pic/cycles_colored.svg){ align=left width="250" }
+        Now the user starts working on the February work cycle. He or she creates two
+        scenarios for the February cycle (one with a low capacity assumption and one with
+        a high capacity assumption). The user can then decide to elect the low capacity
+        scenario as the "official" scenario for February. To accomplish that, he just
+        needs to promote the low capacity scenario as _**primary**_ for the February cycle.
+
+        The tree of entities resulting from the various scenarios created is represented
+        in the picture on the left. The underlined scenarios are _**primary**_.
+
+    !!! note
+
+        Note that cycles are optional. If a scenario has no Frequency, it will not be
```

Comment:
We're already in a Note block

```suggestion
        For a scenario, cycles are optional. If a scenario has no Frequency, it will not be
```